src/wazuh_modules/wmodules.c

```diff
@@ -101,6 +101,10 @@ int wm_config() {
         wm_add(module);
     }
 
+    // Inventory sync
+    if ((module = wm_inventory_sync_read()))
+        wm_add(module);
+
```

Comment:
This will change the order on how the different modules of wmodules will be initialized right? Since the inventory sync is a key player from now on, this was intentionally moved to be initialized first? Is there an issue if not? I would leave a comment in that case